lib/mongoose_push_web/schemas.ex

```diff
@@ -0,0 +1,36 @@
+defmodule MongoosePushWeb.Schemas do
+  alias OpenApiSpex.Schema
+
+  defmodule APIv1Request do
```

Comment:
Let's never do that :)
We should always have 1-to-1 mapping with modules <-> files.